packages/flutter/test/material/mergeable_material_test.dart

```diff
@@ -0,0 +1,170 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('MergeableMaterial merge/separate test', (WidgetTester tester) async {
+    final GlobalKey mmKey = new GlobalKey();
```

Comment:
I think you can remove the mmKey, you're not actually using it as far as I can tell.